uiowa/uiowa
# adjust publish_options_allowed_values

fixes #5740
resolves https://github.com/uiowa/uiowa/issues/5924

This function is only being used for the page content type, so the bundle check is redundant but problematic in instances (generateSampleValues) where the bundle info doesn't exist.

# To Test
Before checking out this branch, go to https://default.uiowa.ddev.site/admin/structure/types/manage/page/display and see the WSOD.

- Checkout this branch. `cr`
- Go to https://default.uiowa.ddev.site/node/add/page and see hide title and no sidebar publish options.
- Go to https://default.uiowa.ddev.site/admin/structure/types/manage/page/display and not WSOD
- Additionally, several other `options_allowed_values` functions were updated to be correct
